disnake/ui/input_text.py

```diff
@@ -65,12 +65,23 @@ class InputText:
         The maximum length of the input text.
     """
 
+    __item_repr_attributes__: Tuple[str, ...] = (
+        "style",
+        "label",
+        "custom_id",
+        "placeholder",
+        "value",
+        "required",
+        "min_length",
+        "max_length",
+    )
+
     def __init__(
         self,
         *,
-        style: InputTextStyle,  # Default to InputTextStyle.short?
         label: str,
         custom_id: str,
+        style: InputTextStyle = None,
```

Comment:
Since this is now `short` by default, why not doing `style: InputTextStyle = InputTextStyle.short` for various reasons:

1. We would avoid confusions in both the documentation and in the code (mostly the documentation)
2. No need to use `style or InputTextStyle.short`

Well only that but you get the point :p

Reply:
Oops, my hands assigned `None` automatically. Should've used my brain